src/ts/client/grapesjs/settings.ts

```diff
@@ -59,7 +59,6 @@ export const settingsDialog = (editor, opts) => {
         editor.trigger(ClientEvent.SETTINGS_SAVE_END, page)
         editor.stopCommand(cmdOpenSettings)
       }
-      form.querySelector('input')?.focus()
```

Comment:
actually this makes sense? will it focus the first input ?

Reply:
I removed it because it was in conflict with the automatic focus of the primary button of the modal, but maybe you only wanted this for dialogs? If so, I'll remove this behaviour and rollback this line.

Reply:
yes this is better to focus the first input, as it is in a form. So enter will submit the form, esc cancel, tab go to next input